### PR TITLE
feat: token model v2

### PR DESCRIPTION
**Pull request by ccamel**
Specification of the token model v2 for the OKP4 protocol by @tpelliet.

image

## Summary by CodeRabbit

- **Documentation**
  - Updated the token model documentation for enhanced clarity and accuracy.
  - Renamed "OKP4 Token Model" to "KNOW Token Model" and refined the description of objectives.

- **Refactor**
  - Improved the calculation logic and data structure for the token model's visualization.

- **Style**
  - Adjusted the caption and data source presentation in the visualization component.

### CODE REVIEW

**Review by MdechampG — APPROVED** [review]
👍🏻

**Review by tpelliet — APPROVED** [review]
👌

File: docs/whitepaper/token-model.tsx [hunk 1]
```diff
@@ -2,40 +2,23 @@ import { ResponsiveLine } from '@nivo/line'
 import React from 'react'
 import { usePrismTheme } from '@docusaurus/theme-common'
 
-const a = 200000000
-const i = 0.0002
-const c = 2.5
-const bt = 0.66
+const c = 3
 
 const lineColor = 'hsl(331, 70%, 50%)' // TODO: use theme color
```
[the remark this answers is not in the document]
Comment (ccamel) [reply docs/whitepaper/token-model.tsx:7]: Not a major consideration in the perspective of this documentation. The code is clear enough as it stands, in my opinion.

**Review by amimart — APPROVED** [review]
LGTM 😉

**Pull request merged**